src/spikeinterface/postprocessing/template_metrics.py

```diff
@@ -197,49 +329,106 @@ def get_trough_and_peak_idx(template):
     """
     Return the indices into the input template of the detected trough
     (minimum of template) and peak (maximum of template, after trough).
-    Assumes negative trough and positive peak
+    Assumes negative trough and positive peak.
```

Comment:
I'm curious about the use of the word peak in this context. Typically we compare trough (min) vs crest (max). In SpikeInterface the user usually has the choice to specify the peak as either 'pos' or 'neg'. So would it be clearer to use the trough and crest to be absolute terms and the user decide on the meaning of peak based on `peak_sign` type arguments in general? This isn't enforced in si, but I'm more just curious.